modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -128,7 +133,68 @@ Starting from Neo4j 2025.01, you can use `existingData`, `seedURI`, `seedConfig`
 The `existingDataSeedInstance` and `existingDataSeedServer` are still not supported with the `CREATE OR REPLACE DATABASE` command.
 More details about seeding options can be found in xref::clustering/databases.adoc#cluster-seed[Seed a cluster].
 ====
+======
+[.include-with-cypher-25]
+======
+The `CREATE DATABASE [OR REPLACE]` command can have a map of options, e.g. `OPTIONS {key: 'value'}`.
+
+[options="header"]
+|===
+
+| Key | Value | Description
+
+|`existingDataSeedServer`
+| ID of the cluster server
+|
+Defines which server is used for seeding the data of the created database.
+The server ID can be found in the `serverId` column after running `SHOW SERVERS`.
+
+| `seedURI`
+| URI to a backup, a folder that contains backup artifacts or a dump from an existing database.
+|
+Defines a seed from an external source which will be used to seed all servers.
+
+| `seedConfig`
+| Comma-separated list of configuration values.
+|
+For more information see xref::clustering/databases.adoc#cluster-seed-uri[Seed from URI].
+
+| `txLogEnrichment`
+| `FULL` \| `DIFF` \| `OFF`
+|
+Defines the level of enrichment applied to transaction logs for Change Data Capture (CDC) purposes.
 
+For details about enrichment mode, see link:{neo4j-docs-base-uri}/cdc/current/get-started/self-managed/#set-enrichment-mode/[Change Data Capture Manual -> Enable CDC on self-managed instances -> Set the enrichment mode].
+
+| `storeFormat`
+| `aligned` \| `standard` \| `high_limit` \| `block`
+|
+Defines the store format if the database created is new.
+`high_limit` and `standard` formats are deprecated from 5.23.
+For more information on store formats, see xref::database-internals/store-formats.adoc[Store formats].
+
+If the store is seeded with `seedURI` or `existingDataSeedServer`, or if the command is used to mount pre-existing store files already present on the disk, they will retain their current store format without any modifications.
+
+| `seedRestoreUntil`
+| Datetime or transaction id. E.g. `datetime("2025-01-01T12:15:00.000+0100")` or `123456`
+| 
+If you are passing a `seedURI` that leads to a backup chain, including differential backups, you can choose to not apply all the transactions in the differential backups.
+To seed up to a specific date, specify a `datetime` and this will seed the database with transactions committed before the provided timestamp.
+To seed up to a specific transaction ID, specify a transaction id, and this will seed the database with transactions up to, but not including the specified transaction.
```

Comment:
```suggestion
To seed up to a specific transaction ID, specify a transaction ID.
This will seed the database with transactions up to, but not including the specified transaction.
```